packages/apollos-ui-kit/src/OverlayBackgroundImage/index.js

```diff
@@ -65,7 +65,8 @@ class OverlayBackgroundImage extends Component {
     width: DeviceWindow.width,
   };
 
-  handleLayout = ({ nativeEvent: { layout } = {} }) => this.setState(layout);
+  handleLayout = ({ nativeEvent: { layout: { width } = {} } = {} }) =>
+    this.setState(width);
```

Comment:
This doesn't seem right, this looks like it would set the whole `state` object to an integer, instead of setting it to `{ width: int } `. What's the goal of this change?

Reply:
My bad. Missing some brackets for shorthand object assignment.
```suggestion
    this.setState({ width });
```

Reply:
Sweet! That's what I was expecting to see

Reply:
The goal is to only set the one layout value we need rather than the entire layout object in state.

Reply:
Yah I figured, makes perfect sense with this change. Good catch.